pypvt/element_fluid_description.py

```diff
@@ -343,63 +343,63 @@ def get_df(self, keyword):
                     "PRESSURE": self.ref_press,
                     "INITRS": self.initrs,
                     "INITRV": self.initrv,
-                    "ACCURACY": self.accuracy,
+                    # "ACCURACY": self.accuracy,
                 },
                 ignore_index=True,
             )
 
         elif keyword == "RSVD":
             df = pd.DataFrame(columns=["KEYWORD", "EQLNUM", "RS", "Z"])
 
-            for i in range(len(self.rsvd_rs)):
+            for (rs, depth) in zip(self.rsvd_rs, self.rsvd_depth):
                 df = df.append(
                     {
                         "KEYWORD": "RSVD",
                         "EQLNUM": self.eqlnum,
-                        "RS": self.rsvd_rs[i],
-                        "Z": self.rsvd_depth[i],
+                        "RS": rs,
+                        "Z": depth,
                     },
                     ignore_index=True,
                 )
 
         elif keyword == "RVVD":
             df = pd.DataFrame(columns=["KEYWORD", "EQLNUM", "RV", "Z"])
 
-            for i in range(len(self.rvvd_rv)):
+            for (rv, depth) in zip(self.rvvd_rv, self.rvvd_depth):
                 df = df.append(
                     {
                         "KEYWORD": "RVVD",
                         "EQLNUM": self.eqlnum,
-                        "RV": self.rvvd_rv[i],
-                        "Z": self.rvvd_depth[i],
+                        "RV": rv,
+                        "Z": depth,
                     },
                     ignore_index=True,
                 )
 
         elif keyword == "PBVD":
             df = pd.DataFrame(columns=["KEYWORD", "EQLNUM", "PB", "Z"])
 
-            for i in range(len(self.pbvd_pb)):
+            for (pb, depth) in zip(self.pbvd_pb, self.pbvd_depth):
                 df = df.append(
                     {
                         "KEYWORD": "PBVD",
                         "EQLNUM": self.eqlnum,
-                        "PB": self.pbvd_pb[i],
-                        "Z": self.pbvd_depth[i],
+                        "PB": pb,
+                        "Z": depth,
                     },
                     ignore_index=True,
                 )
 
         elif keyword == "PDVD":
             df = pd.DataFrame(columns=["KEYWORD", "EQLNUM", "PD", "Z"])
 
-            for i in range(len(self.pdvd_pd)):
+            for (dp, depth) in zip(self.pdvd_pd, self.pdvd_depth):
```

Comment:
Use `pd` as variable?

Reply:
Point taken! Variable naming in pypvt is confusing and inconsistent. I'll perform a cleanup.